vignettes/BiocCheck.Rmd

```diff
@@ -550,10 +550,48 @@ The output of this check includes the first 6 offending lines of code;
 see more with `BiocCheck:::checkFormatting("path/to/YourPackage",
 nlines=Inf)`.
 
-There is a helpful package [formatR][formatR] that can be used for formatting of
-R code to particular coding standards.
+There are several helpful packages that can be used for formatting of
+R code to particular coding standards such as  [formatR][formatR] and
+[styler][styler] as well as the "Reformat code" button in
+[RStudio Desktop][RStudio]. Each solution has its advantages, though 
+[styler][styler] works with `roxygen2` examples and is actively
+maintained. You can re-format your code using [styler][styler] as shown
+below:
+
+```{r reformat_code, eval = FALSE}
+## Install styler if necessary
+if (!requireNamespace("styler", quietly = TRUE)) {
+    install.packages("styler")
+}
+## Automatically re-format the R code in your package
+styler::style_pkg(transformers = styler::tidyverse_style(indent_by = 4))
+
+## If using biocthis:
```

Comment:
This too is problematic, with biocthis not yet in Bioconductor. I also feel that, even if biocthis were in Bioconductor, it seems likely that we would like for it to have been tested and used 'for a while' before sending users there immediately.